lib/rucio/db/sqla/models.py

```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
```

Comment:
Please add the changes (new index and new column in requests table) to oracle sql file as well
https://github.com/rucio/rucio/blob/master/etc/sql/oracle/schema.sql